jquery.scrolly.js

```diff
@@ -40,16 +40,11 @@
     this.velocity = this.$element.attr('data-velocity');
     this.bgStart = parseInt(this.$element.attr('data-fit'), 10);
 
-    $(document).scroll(function(){
+   $(document).bind('scroll', function(){
```

Comment:
I think `bind` is deprecated, no? Maybe I'm wrong, but we should try to use `on` instead.